src/index.js

```diff
@@ -2,5 +2,5 @@
 
 export {default} from './command'
 export {default as Topic} from './topic'
-export {BooleanFlag, StringFlag, NumberFlag, Flag} from './flags'
+export {boolean, string, number, app, remote, Flag} from './flags'
```

Comment:
this exports them as 'boolean, string' etc. What I want is all of these grouped under 'flags':

```js
import {boolean, string, number, app, remote, Flag} from './flags'
export const flags = {
  boolean,
  string,
  number,
  remote,
  app,
  Flag
}
```

there might be a more concise way to write it